internal/schedulingNodeOperator/schedulingNodeFromNode.go

```diff
@@ -75,18 +74,10 @@ func (r *SchedulingNodeReconciler) createSchedulingNode(ctx context.Context, nod
 func (r *SchedulingNodeReconciler) setNeighborsFromAdv(sn *v1alpha1.SchedulingNode, ctx context.Context, node corev1.Node) error {
 	var adv advv1.Advertisement
 
-	namespacedNodeName := types.NamespacedName{
-		Name:      strings.Join([]string{"advertisement", node.Annotations["cluster-id"]}, "-"),
-		Namespace: "default",
-	}
+	advname := strings.Replace(node.Labels["kubernetes.io/hostname"], "vk-", "", 1)
```

Comment:
I don't know if this affects the operator, but the names of the virtual nodes have been changed to `liqo-<clusterID>`

Reply:
It may be, what is the naming for the advertisement?